### DISCUSSION TITLE
Upcoming Event: Terraform AWS Modules Discussion - August 31st

### DISCUSSION THREAD

**Discussion by chiche-ds** [chiche-ds/30-Day-Terraform-challenge-#133] [General]
Hi Team,

I’m excited to announce that Anton Babenko will lead a Terraform AWS Modules discussion on August 31st. This session will demonstrate what Terraform AWS Modules are, how to use them effectively, and best practices. Anton will also be available to answer questions and engage in an open discussion.

To ensure that the session meets your needs and expectations, please share any specific topics or questions you’d like covered. Your feedback will help us make this event as valuable as possible.

Let's hear your thoughts in the comments

**Comment by kevshakes** [comment]
My questions to Anton: 

1. What does it take to maintain one of the most used modules on the Terraform Registry?
2. What are some of the features and improvements coming up on the AWS Modules and Terraform in general?
